lib/pf/condition/matches.pm

```diff
@@ -0,0 +1,60 @@
+package pf::condition::matches;
+=head1 NAME
+
+pf::condition::matches
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::condition::matches
+
+=cut
+
+use strict;
+use warnings;
+use Moose;
+extends qw(pf::condition);
+
+has value => (
+    is => 'ro',
+    required => 1,
+    isa  => 'Str',
+);
+
+sub match {
+    my ($self,$arg) = @_;
+    my $regex = $self->value;
+    return 0 if(!defined($arg));
+    return $arg =~ /$regex/;
```

Comment:
As we discussed in the violation PR, this should be a pure match, not regex.
Unless you want to leave it as is